### PR TITLE
Minor change to wshfl.c.

### PR DESCRIPTION
**Pull request by sidward**

### CODE REVIEW

File: src/noise.c [hunk 1]
```diff
@@ -69,6 +73,7 @@ int main_noise(int argc, char* argv[])
 
 	float stdev = sqrtf(var);
 
+	#pragma omp parallel for
 	for (long i = 0; i < T; i++) {
```
Comment (uecker) [inline src/noise.c]: This should be two separate pull requests. The first change is perfectly fine, but the second isn't as it makes the noise command non-deterministic.

Comment (sidward) [reply src/noise.c]: Hi Martin,

Sounds good. I'll revert this pull to just the scaling change.
